flycheck.el

```diff
@@ -7138,7 +7138,7 @@ https://github.com/rust-lang/rust/blob/master/src/librustc_errors/json.rs#L154"
     ;; If there are no spans, the error is not associated with a specific
     ;; file but with the project as a whole.  We still need to report it to
     ;; the user by emitting a corresponding flycheck-error object.
-    (unless spans
+    (unless (or spans (null error-code))
```

Comment:
I'm not sure I understand this check (I trust that it's correct. I'm just curious ^^)

Reply:
I've replace it with `(and error-code (not spans))`, I hope it makes it a bit more clear